src/schema/v2/show.ts

```diff
@@ -338,14 +351,18 @@ export const ShowType = new GraphQLObjectType<any, ResolverContext>({
       exhibitionPeriod: {
         type: GraphQLString,
         description: "A formatted description of the start to end dates",
-        resolve: ({ start_at, end_at }) => dateRange(start_at, end_at, "UTC"),
-      },
-      shortExhibitionPeriod: {
-        type: GraphQLString,
-        description:
-          "A formatted description of the start to end dates with abbreviated months",
-        resolve: ({ start_at, end_at }) =>
-          dateRange(start_at, end_at, "UTC", true),
+        args: {
+          format: {
+            type: ExhibitionPeriodFormatType,
+            description:
+              "SHORT or LONG, SHORT returns a shorter exhbition period description with months abbreviated, defaults to LONG",
+          },
+        },
+        resolve: ({ start_at, end_at }, args) => {
+          const { format } = args
+          const useShortFormat = format === "SHORT"
+          return dateRange(start_at, end_at, "UTC", useShortFormat)
```

Comment:
**suggestion**: Related to the comment above, if `format` is always either `LONG` or `SHORT` and never null, we could pass that variable directly through to `dateRange` and let that function decide what to do.